templates/resources/index.html

```diff
@@ -0,0 +1,405 @@
+{% extends "base_layout.html" %}
+
+{% block title %}Resources{% endblock %}
+
+{% block meta_description %}A collection of useful and educational material and information about MicroK8s. Learn more about Kubernetes from some of the industry’s best.{% endblock meta_description %}
+
+{% block meta_copydoc %}https://docs.google.com/document/d/1PB5-CamGmSjhbuyuGSariC9zUW-8krAJD0l8JyvNk20/edit#{% endblock %}
+
+{% block content %}
+
+<section class="p-strip--suru">
+  <div class="row u-equal-height">
+    <div class="col-7">
+      <h1>Understand MicroK8s: Resources</h1>
+      <p>Explore a collection of useful resources that will help you understand and evaluate MicroK8s when building your Kubernetes cluster.</p>
+      <p>Whitepapers, blogs, customer presentations, webinars, and more are conveniently collected here for you. We’ll guide you through what each resource is about, and add more to this page over time.</p>
+    </div>
+  </div>
+</section>
+<section class="p-strip">
+  <div class="u-fixed-width">
+    <h2>Videos and webinars</h2>
+  </div>
+  <article class="p-strip is-shallow">
+    <div class="row">
+      <div class="col-4">
+        <h3 id="what-is-microk8s" class="p-heading--4">What is MicroK8s?</h3>
+      </div>
+      <div class="col-4">
+        <p>This 451 Research whitepaper focuses on the key factors that drive the evolution of IoT edge and the requirements around which Kubernetes need to optimise to become the standard edge platform.</p>
```

Comment:
I think maybe this change in the doc was made after you already started working on this, but according to the doc it's supposed to read: "This super short video gives an overview of MicroK8s: A lightweight, zero-ops, pure upstream Kubernetes which is at home on your laptop, in CI workflows, on a cloud or even embedded in your IoT Edge production environment.


Watch it [here](https://www.youtube.com/watch?v=v9KI2BAF5QU)."